src/main/java/com/aws/greengrass/util/Exec.java

```diff
@@ -89,6 +89,11 @@
     private Copier stderrc;
     private Copier stdoutc;
 
+    protected Exec() {
+        environment = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
```

Comment:
add a comment for why we're doing this. Also should we apply this to Windows only?

Reply:
Fixed. Yes only Windows should be case-insensitive